### DISCUSSION TITLE
Mobs/Animals Health Bar

### DISCUSSION THREAD

**Discussion by JustAr1s** [BedrockTweaks/Files#152] [Ideas]
Add a new **Health bar** intop of mobs and animals heads.

**Comment by AtlasOfEden** [comment]
I'd also like to see this!

**Comment by IdotIcom** [comment]
https://modbay.org/textures/522-simple-health-bar.html

**Reply by AtlasOfEden** [reply]
I have a separate pack for Health Bars, but I wanted one integrated with all the other great stuff on BedrockTweaks.com, that's the only reason I upvoted this.

**Comment by SquatchHunter** [comment]
these kinds of packs are probably out of scope for Bedrock Tweaks as Bedrock Tweaks aims to be a more Vanilla+ style of pack, something like health bars would be a "Lite" mod imho
